README.md

```diff
@@ -31,7 +31,8 @@ _NOTE:_ We use `yarn` dependency manager. To install it, follow its [documentati
 -   `--no-install` skip yarn installing the app dependencies
 -   `--verbose` output additional logs, useful mainly for debugging and bug reports
 -   `-h, --help` output usage information
--   `--backend <backend>` switch backend to a desired one (default: bear)
+-   `--backend <backend>` switch backend to a desired one (default: `bear`)
+-   `--flavor <flavor>` desired flavor of the application: `ts` for TypeScript, `js` for JavaScript (default: `ts`)
```

Comment:
let's discuss this. as much as i'd personally prefer default to be TS the JS may be a better default. it is also compatible with how the toolkit worked previously

Reply:
Fixed.